.gitignore

```diff
@@ -18,9 +18,8 @@ phpunit.xml
 /js/base.js
 /templates/components/
 /oe_theme/
-*.tar.gz
```

Comment:
The "create release" command generates tar balls, hence the line it, we should not remove it as it is useful when testing releases.